src/index.js

```diff
@@ -1,59 +1,121 @@
-// import _ from 'lodash';
+/* eslint no-use-before-define: */
+
```

Comment:
- [ ] Please do not disable linters as they are there to help you stick to a strict coding style 👍

Reply:
Hi @codepantha. Thank you for your positive feedback. The removeTask() function calls showTask() function again the showTask() function calls removeTask() function. That said, I could not get linter-free JavaScript code when I interchange the position of the two functions. That is why I disabled the linter for this case . Thanks for your reply.

Reply:
Try to move the `showTask()` function definition before it is called in the event listener which you added to the `addBtn `on line 36. I hope this helps